queries.sql

```diff
@@ -8,3 +8,29 @@
  SELECT * FROM animals WHERE neutered = TRUE;
  SELECT * FROM animals WHERE name <> 'Gabumon';
  SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3; 
+
+ BEGIN ;
+ SET species = 'unspecified' ;
+ ROLLBACK ;
+ UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon' ;
+ UPDATE animals SET species = 'pokemon' WHERE species IS NULL ;
```

Comment:
👍 You're on the right track, but there are a few adjustments to consider:

- [ ] It appears you've combined two transactions into one. As per the project requirements, there should be two separate transactions. Let's address this by separating the transactions.
![image](https://github.com/AbuTalha3/vet-clinic-database/assets/55840999/644fc62c-b452-4d4b-bf5d-0b7a03aa6a09)
- [ ] In the first transaction, you should:
    - Update the "animals" table by setting the "species" column to "unspecified" inside a transaction.
    - Verify that the change was made within the transaction.
    - Roll back the change.
    - Verify that the "species" column went back to the state before the transaction. Please ensure this sequence is followed.

- [ ] In the second transaction, you should:
    - Update the "animals" table by setting the "species" column to "digimon" for all animals with names ending in "mon."
    - Update the "animals" table by setting the "species" column to "pokemon" for all animals that don't already have a species set.
    - Verify that these changes were made within the transaction.
    - Commit the transaction.
    - Verify that the changes persist after the commit. Please ensure this sequence is followed.

- [ ] After updating the queries to meet the project requirements, please don't forget to update the screenshots in the PR Description if necessary. This will help reviewers and team members understand the changes more clearly.

Your dedication to this project is commendable! Keep up the good work. If you have any questions or need further guidance, feel free to ask. 👍🔄📊

Reply:
- First transaction ... 

![image](https://github.com/AbuTalha3/vet-clinic-database/assets/127955219/bb5dc57f-3675-4888-ae5e-8a1dd20baa16)

- Second transaction

![image](https://github.com/AbuTalha3/vet-clinic-database/assets/127955219/4ec43090-be6b-49ea-81fc-2b8129c0acfb)

![image](https://github.com/AbuTalha3/vet-clinic-database/assets/127955219/a6936eb6-5852-46f6-bd47-d9c9b9f2bb22)